server/src/utils/validations/signIn.js

```diff
@@ -0,0 +1,20 @@
+import isEmpty from './isEmpty';
+
+function validateRegisterInput(user) {
```

Comment:
for both signin and signup validations, you can create a common wrapper, which will take object, key as params and will return errors object.